src/renderer/helpers/playlists.js

```diff
@@ -31,6 +35,10 @@ export function getSortedPlaylistItems(playlistItems, sortOrder, locale, reverse
   })
 }
 
+export function checkDurationFromVideo(a) {
+  return (isNaN(a.lengthSeconds) || a.lengthSeconds === 0 || typeof a.lengthSeconds !== 'number') ? 0 : a.lengthSeconds
+}
```

Comment:
No export on `videoDurationWithFallback` unless it's used outside
```suggestion
export function videoDurationPresent(v) {
  if (typeof v.lengthSeconds !== 'number') { return false }
  
  return !(isNaN(v.lengthSeconds) || v.lengthSeconds === 0)
}

function videoDurationWithFallback(v) {
  if (videoDurationPresent(v)) { return v.lengthSeconds }

  // Fallback
  return 0
}
```